### ISSUE TITLE
Collaborator access

### ISSUE THREAD

**Issue by Richardant** [ngraves95/attacktimer#68]
@ngraves95 Hey Nick, can you give me collaborator access? So i can help you with this repo, you seem like a busy guy 😅

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
https://github.com/ngraves95/attacktimer/pull/70 @ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
https://github.com/ngraves95/attacktimer/pull/70 @ngraves95 take a look here

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
Sir leagues is just around the corner, are you going to keep this updated or do you want some help? @ngraves95

**Comment by Richardant** [comment]
Nick! Leagues is just around the corner, are you going to keep this updated or do you want some help? @ngraves95

**Comment by ngraves95** [comment]
I will review and merge PRs as my real-life schedule allows. I would be appreciative if you took a stab at extending the old leagues code to work for this upcoming league.

**Comment by Richardant** [comment]
> I will review and merge PRs as my real-life schedule allows. I would be appreciative if you took a stab at extending the old leagues code to work for this upcoming league.

Sure, mate, I'm happy to help with the leagues. Could I help you with the PRs if you provide me with collaborator access? That will also allow for faster updates and fixes since I'm very active here.

**Comment by Richardant** [comment]
> I will review and merge PRs as my real-life schedule allows. I would be appreciative if you took a stab at extending the old leagues code to work for this upcoming league.

Sure, mate, I'm happy to help with the leagues. Could I help you with the PRs if you provide me with collaborator access? That will also allow for faster updates and fixes since I'm very active here. @ngraves95

**Comment by Richardant** [comment]
Do not forget to do push the more recent changes from #70 to https://github.com/runelite/plugin-hub/pull/6993

it's been more than a month since my PR and it still hasn't been merged into the phub repo 😢 @ngraves95

**Comment by Richardant** [comment]
@ngraves95

**Comment by Richardant** [comment]
@ngraves95 #72

**Comment by Richardant** [comment]
@ngraves95

**Comment by Lexer747** [comment]
@Richardant please respect a persons time and notifications. I'm not sure tagging everyday is appreciated - have some patience, this repo doesn't need to move as fast as runelite or the plugin hub. 

In terms of leagues, I'm not actually playing them (just watching) but I assume PRs would always be welcome if/when you reach the tiers where the attack speeds start to change.

**Comment by Richardant** [comment]
> @Richardant please respect a persons time and notifications. I'm not sure tagging everyday is appreciated - have some patience, this repo doesn't need to move as fast as runelite or the plugin hub.
> 
> In terms of leagues, I'm not actually playing them (just watching) but I assume PRs would always be welcome if/when you reach the tiers where the attack speeds start to change.

Sorry @Lexer747, but this is something that is practically required to be able to update in this repository since the author is practically inactive, and only after "tagging" him does he respond. My last PR took 1 month to finally be merged into the plugin hub repository because @ngraves95 takes too long between responses. Adding someone else as a collaborator (either you or me) would be very helpful to speed up this repository and not wait so long for a simple update (the PR #72 has already been pending for a week).
